automation/test.sh

```diff
@@ -73,7 +64,7 @@ fi
 
 if [[ $TARGET =~ sriov.* ]]; then
   export KUBEVIRT_NUM_NODES=3
-  export KUBEVIRT_DEPLOY_CDI=false
+  export KUBEVIRT_DEPLOY_CDI="false"
```

Comment:
the usage of with or without quotes is not consistent (in this file, and cross files)
would worth please to align it
(line 34 and up in this file, and there are more files that have it with and without quotes)

Reply:
As the usage was inconsistent before already, I'm not sure that this PR should be blocked by this. We can of course do this in a follow up PR. WDYT?

Reply:
ok